### ISSUE TITLE
[Adaptive Card > Image Gallary.Json]: Images under cards are not accessible through the swipe gesture.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8579]
### Target Platforms

iOS

### SDK Version

1.0 (2.4.0-beta.23.6.11.1)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Device: iPhone 11
iOS Version: 16.5.1
Screen Reader: Voice Over

**Repro Steps:**
1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Navigate to "V1.5" control and select it.
4. Navigate on "V1.5" screen and select "Scenarios" button.
5. 'Scenario' screen will appear.
6. Navigate to "Image Gallary.json" control and double tap to activate it.
7. Perform right swipe and observe the issue.

**Actual Result:**
While navigating under the 'Image Gallery' card, Images under cards are not accessible through the swipe gesture.

**Expected Result:**
Images should be accessible through the swipe gesture when user navigates the images gallery.

**Note:**
Same issue is repro throughout the application for the images.

**Observation:**
This issue is working fine while navigating using keyboard.

**User Experience:**
It will impact the screen reader user as they will not be able to navigate to the images and also do not perform the desirable action what they want to do on the card.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/meaningful-sequence

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “

### Screenshots

https://github.com/microsoft/AdaptiveCards/assets/95913748/24e49d01-e66f-49ba-b9ba-6c7a832ce53d

### Card JSON

```JSON
NA
```

**Comment by vagpt** [comment]
Now tracking with below URL.

https://github.com/microsoft/AdaptiveCards-Mobile/issues/43
